### ISSUE TITLE
FreeSwitch crashes after starting a recording

### ISSUE THREAD

**Issue by jpawlowski** [amooma/GS5#17]
When I am connected to a conference call and push the button to record the session, FreeSwitch crashes and needs to be restarted.
Following lines are logged:

```
2012-12-19 12:54:25.446371 [DEBUG] switch_core_session.c:975 Send signal sofia/gemeinschaft/e91e1047f66f3fbe8cd4@192.168.6.60 [BREAK]
2012-12-19 12:54:25.466397 [ERR] switch_core_file.c:80 File [/var/lib/freeswitch/recordings//] is a directory not a file.
2012-12-19 12:54:25.466397 [DEBUG] switch_core_media_bug.c:502 Attaching BUG to sofia/gemeinschaft/e91e1047f66f3fbe8cd4@192.168.6.60
2012-12-19 12:54:25.466397 [NOTICE] sofia.c:7417 Recording sofia/gemeinschaft/e91e1047f66f3fbe8cd4@192.168.6.60 to /var/lib/freeswitch/recordings//
2012-12-19 12:54:25.466397 [DEBUG] sofia.c:7448 dispatched freeswitch event for INFO
2012-12-19 12:54:25.466397 [DEBUG] switch_core_session.c:975 Send signal sofia/gemeinschaft/e91e1047f66f3fbe8cd4@192.168.6.60 [BREAK]
```

**Comment by wintermeyer** [comment]
It is not a nice bug. But I will not loose sleep over this. Worst case: We'll fix it with a 5.0.1 release.

**Comment by jpawlowski** [comment]
I thought so ;)

**Comment by spag** [comment]
This bug does not affect install.sh installations.

Fixed in commit 6bf23a88899522a52740113a3cfdccc7ae533fca

**Comment by jpawlowski** [comment]
install.sh is not an argument ;-p
GBE uses the official Debian packages (although they are build on our own) so we can assume this is the official behavior of FS. You may also have an old FS version in your install.sh environment, we are using the FS master head branch as suggested by the FS team.
